assignments/callbacks.js

```diff
@@ -4,12 +4,19 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 /* 
 
+*/
   //Given this problem: 
   
   function firstItem(arr, cb) {
-    // firstItem passes the first item of the given array to the callback function.
+// firstItem passes the first item of the given array to the callback function.
+    return cb(arr[3])
+    
```

Comment:
So you are properly using the call back here, but you are targeting the index of  `3`  (position 4 in the array since arrays start at 0).